python/lsst/ip/isr/isrTask.py

```diff
@@ -859,9 +857,21 @@ class IsrTaskConfig(pipeBase.PipelineTaskConfig,
     # Vignette correction configuration.
     doVignette = pexConfig.Field(
         dtype=bool,
-        doc="Apply vignetting parameters?",
+        doc="Compute and attach validPolygon to exposure according to vignetting parameters?",
```

Comment:
It's not that clear here if a "validPolygon" is some sort of formal type, or if it is defined as something else. I know we should try to be concise here, but if possible, perhaps you could add a few more phrases to explain what "validPolygon" is for someone who looks at this doc.